code/dqc/symfilegrowth.q

```diff
@@ -0,0 +1,16 @@
+\d .dqc
+
+/- Take the average symfile count from the past n days, then check that todays
+/- sym file count hasn't grown more than pct%.
+symfilegrowth: {[directory;ndays;pct]
+  /- Get handles to the DQE and the DQEDB. 
+  h:first exec w from .servers.SERVERS where proctype=`dqedb;
+  / Make sure we have enough days in the dqedb.
+  if[ndays>h"count .Q.pv";:(0b;"ERROR: number of days (ndays) exceeds number of available dates")];
+  / Get todays sym file count.
+  tc:dqeh"first exec resvalue from .dqe.resultstab where funct=`symcount";
+  /- Get average sym file count from previous days, where select statement is
+  /- "select from resultstab where date>=.z.d-ndays,funct=`symfilecheck"
+  ac:exec avg resvalue from dqedbh(?;`resultstab;(((';~:;<);`date;(-;`.z.d;ndays));(=;`funct;enlist`symcount));0b;());
```

Comment:
Should be able to use a `select` statement here, looks like the only variable parameter is `ndays`.

Reply:
So I could just do
```q
dqedbh "select from resultstab where date>=.z.d-",string[ndays],",funct=`symfilecheck"
```
?

Reply:
Looks good, easier to follow that way.